manifests/init.pp

```diff
@@ -328,6 +331,7 @@
   Boolean                    $trap_service_enable          = false,
   Boolean                    $trap_service_hasstatus       = true,
   Boolean                    $trap_service_hasrestart      = true,
+  String[1]                  $service_restart              = "/bin/systemctl reload $service_name",
```

Comment:
```suggestion
  String[1]                  $service_restart              = "/bin/systemctl reload ${service_name}",
```

Reply:
Also this will break (at least) FreeBSD support. This parameter should probably be optional, default to `undef` and set to this value only on systems where it makes sense through Hiera.